.github/workflows/intl.yml

```
@@ -42,3 +43,7 @@ jobs:
 
       - run: dart test
         if: ${{matrix.run-tests}}
+
+      - name: Run Chrome tests - wasm
+        run: dart test --platform chrome --compiler dart2wasm
+        if: always() && steps.install.outcome == 'success' && matrix.sdk == 'dev'
```

```suggestion
        if: always() && matrix.sdk == 'dev'
```

and drop the `install` ID above.